applications/polkadot-kusama-apk-bridge-poc.md

```diff
@@ -0,0 +1,155 @@
+
+Our approach with APK proofs offers a novel and potentially more efficient alternative focusing on scalability while demonstrating the capabilities of Web3Sum protocol developed by Web3 foundation. The `w3f/apk-proofs`, `w3f/ring-proofs` and `w3f/bls-beefer` repositories indicate ongoing research and development in this area, and our project aims to build upon and contribute to these efforts.
+
+## Team
+
+*   **Team Name:** Lipika Labs
+*   **Contact Name:** Kumar Gunjan
+*   **Contact Email:** gunjan.cn@gmail.com
+*   **Website:** https://github.com/k-gunjan
+
+### Team members
+
+*   Kumar Gunjan
+
+#### LinkedIn Profiles
+
+*   Kumar Gunjan: https://www.linkedin.com/in/gunjan321/
+
+#### Team Code Repos
+
+- Kumar Gunjan: https://github.com/k-gunjan
+
+### Team's experience
+
+Kumar Gunjan: I bring 15 years of engineering experience spanning Telecom, Distributed system and Cyber security with deep specialization in blockchain development and the Polkadot ecosystem. I am a graduate of the Polkadot Blockchain Academy (Hong Kong cohort). I have demonstrated expertise through multiple successful Polkadot SDK-based projects, including Kylix Finance—delivered under the Web3 Foundation's Decentralized Futures Grant. My technical contributions span the full stack of decentralized infrastructure: I have architected an IPFS-based storage network, designed a zero-knowledge proof protocol, and contributed directly to the Polkadot ecosystem through merged PRs ([polkadot-sdk#3350](https://github.com/paritytech/polkadot-sdk/pull/3350), [substrate-front-end-template#270](https://github.com/jimmychu0807/substrate-front-end-template/pull/270)). My prior experience developing production-grade cross-chain bridges positions me uniquely to deliver the APK Bridge with the security, reliability, and architectural rigor this project demands.
+
+## Development Status
+
+This project is currently in the research and design phase. We have conducted initial investigations into APK proofs, BLS signatures, and existing Polkadot/Kusama infrastructure components like off-chain workers and AssetHub smart contracts. We have identified `w3f/apk-proofs` and `w3f/bls-beefer` as key reference repositories for our implementation. Work in progress can be seen in a PR to `w3f/apk-proofs` [PR-47](https://github.com/w3f/apk-proofs/pull/47). Our prior work on a Web3 Foundation grant and contributions to the Polkadot SDK provide a strong foundation for this project.
+
+## Development Roadmap
+
+This project will be executed in three distinct milestones, each with clearly defined deliverables and verification steps. We anticipate the total duration of the project to be approximately 3 months.
+
+### Overview
+
+*   **Estimated Duration:** - 3 Months
+*   **Full-Time Equivalent (FTE):** - 2.25 FTE
```

Comment:
```suggestion
*   **Estimated Duration:** - 3 Months
*   **Full-Time Equivalent (FTE):** - 0.75 FTE
```

The total duration of the milestones is 1 month and five weeks, with FTE ranging from 0.5 to 1. If you mean it takes one person 2.25 months over the span of 3 months, then it'd be 3 months, 0.75 FTE. Alternatively, you can just write 2.25 months, 1 FTE.